dev-notes/minimal-os-for-fobnail.md

```diff
@@ -0,0 +1,203 @@
+# Minimal OS for Fobnail project
+
+The Fobnail project aims to provide a reference architecture for building
+offline integrity measurement servers on the USB device and clients running in
+Dynamically Launched Measured Environments (DLME). It should allow the Fobnail
+owner to verify the trustworthiness of the running system before performing any
+sensitive operation.
+
+According to the information provided at TrenchBoot Summit 2021 (the fragment
+discussed here - https://youtu.be/xZoCtNV8Qs0?t=5062), the minimum OS, together
+with the connected Fobnail Token, should allow determining the state of the
+platform before the target OS is launched. As we can see in the presentation,
+`heads` would be responsible for the platform's trust control, while in this
+document, we want to present the results of research on the possibilities of
+using various operating systems.
+
+## Different OSs propositions
+
+The research effect is presented below. 4 systems were considered:
+
+* Zephyr
+* Xous
+* seL4
+* Linux
```

Comment:
Linux could be considered here in various forms:
- Heads (busybox user space)
- LinuxBoot (Go user space)
- Linux + Yocto built user space :thinking: ?

Reply:
@pietrushnic would you like those options also be analyzed in this report? In similar way to Linux + Yocto that we already present?

Reply:
@TomaszAIR yes, especially that those can be also viable options and we may use this report for extending scope of the project in future if needed.

Reply:
@pietrushnic I added Busybox and Go sections